kaitai_utils.py

```diff
@@ -40,9 +40,10 @@ def get_plugin_title_from_metadata():
 
 
 class JsonWriter:
-    def __init__(self, writer: BufferedWriter, trace: ExtractionTrace):
+    def __init__(self, writer: BufferedWriter, trace: ExtractionTrace, length: int):
         self.writer = writer
         self.trace = trace
+        self.length = length
```

Comment:
Could you pick a more specific name for `length`? Maybe something that explains that this is the maximum length of a byte array that will be printed as a hex string.

Reply:
Fixed!